int32 pwm_flt
int32 pwm_frt
int32 pwm_rlt
int32 pwm_rrt
int32 pwm_flb
int32 pwm_frb
int32 pwm_rlb
int32 pwm_rrb

bool is_timed

int64 duration